packages/markers/src/browser/problem/problem-preferences.ts

```diff
@@ -21,27 +21,33 @@ export const ProblemConfigSchema: PreferenceSchema = {
     'type': 'object',
     'properties': {
         'problems.decorations.enabled': {
-            'type': 'boolean',
-            'description': 'Show problem decorators (diagnostic markers) in tree widgets.',
-            'default': true,
+            type: 'boolean',
+            description: 'Show problem decorators (diagnostic markers) in tree widgets.',
+            default: true,
         },
         'problems.decorations.tabbar.enabled': {
-            'type': 'boolean',
-            'description': 'Show problem decorators (diagnostic markers) in the tab bars.',
-            'default': true
+            type: 'boolean',
+            description: 'Show problem decorators (diagnostic markers) in the tab bars.',
+            default: true
         },
         'problems.autoReveal': {
-            'type': 'boolean',
-            'description': 'Controls whether Problems view should reveal markers when file is opened.',
-            'default': true
+            type: 'boolean',
+            description: 'Controls whether Problems view should reveal markers when file is opened.',
+            default: true
+        },
```

Comment:
@seantan22 do you mind reverting these changes, they are unrelated to the pull-request.